tox.ini

```diff
@@ -17,6 +17,7 @@ deps =
     .[test]
 conda_deps =
     readdy
+    numpy
```

Comment:
changes reviewed in [PR to this branch](https://github.com/simularium/simulariumio/pull/95)